README.md

```diff
@@ -122,11 +128,26 @@ pieces create
 
 ##### Edit an existing asset:
 
-This currently only works for an assets's name
+This will edit the name and reclassify the current asset.
 
 ```bash
 pieces edit
 ```
+This is used to edit both the classification and name of an asset.
+
+
+```bash
+pieces edit --name "new name"
+```
+to edit the name
+
+```bash
+pieces edit --classification python
+```
```

Comment:
these are looking good can we add a test for these examples w/ the edit command

Reply:
sure I will add add a test for the edit command. There is currently a test for the edit